docs/pages/2019_MARVEL_Psik_MaX/sections/setup.rst

```diff
@@ -20,53 +20,75 @@ The software on this VM is based on `Quantum Mobile
 pre-configured AiiDA installation as well as some test data for the tutorial.
 
 .. note::
-  
-   If you decide to work in pairs, one of you should forward their credentials
-   to the other and you should both use the same IP address and ssh key. 
-   Since you will be sharing the VM and user account, be careful not to delete
-   the work of your colleague.
+   If you decide to work in pairs, one of you should forward their
+   credentials to the other and you should both use the same IP address and
+   ssh key. Since you will be sharing the VM and user account, be careful not
+   to delete the work of your colleague.
 
 Linux and MacOS
 ~~~~~~~~~~~~~~~
 
+It's recommended for you to place the ssh key you received in a folder
+dedicated to your ssh configuration, to do so:
+
 -  If not already present, create a ``.ssh`` directory in your home
    (``mkdir ~/.ssh``), and set its permissions: ``chmod 700 ~/.ssh``
 
--  Copy the two keys ``aiida_tutorial_NUM`` and ``aiida_tutorial_NUM.pub`` 
-   in the ``~/.ssh`` directory 
+-  Copy the two keys ``aiida_tutorial_NUM`` and ``aiida_tutorial_NUM.pub``
+   in the ``~/.ssh`` directory
 
 -  Set the correct permissions on the private key:
-   ``chmod 600 ~/.ssh/aiida_tutorial_NUM``.
-   You can check check with ``ls -l`` that the permissions of this file are now ``-rw-------``.
+   ``chmod 600 ~/.ssh/aiida_tutorial_NUM``. You can check check with ``ls -l``
+   that the permissions of this file are now ``-rw-------``.
+
+After that ssh key is in place, you can go two routes, the first through an
+ssh configuration file specific to the IP address you are going to connect, or,
+using a command in your shell with several configuration options.
```

Comment:
To keep things short, I would scratch that extra explanatory text.
Just go with the `config` option by default and add your "single line version" in a `.. note::` at the end

Reply:
I made it all shorter, I'd like to keep both options at the same level tho.

Reply:
In a tutorial context, it's preferable to keep one preferred path - wherever we start adding multiple ones, we are making our life as tutors harder.

I'm fine with adding the one-line command in a `.. note::` at the end (perhaps also with some hint concerning why people might want to use this), but I would prefer to keep the remaining structure as-is